### ISSUE TITLE
All `have_mark_down_documentation` tests fail

### ISSUE THREAD

**Issue by Corniel** [dotnet-project-file-analyzers/dotnet-project-file-analyzers#285]
All cases of `Design_specs.Rules.have_mark_down_documentation(Rule)` fail. 

``` C#
 [TestCaseSource(nameof(AlleRules))]
 public async Task have_mark_down_documentation(Rule rule)
 {
     using var client = new HttpClient();
     var url = rule.HelpLinkUri.Replace(".html", ".md");
     var response = await client.GetAsync(url);
     response.Should().HaveStatusCode(HttpStatusCode.OK, url);

     var content = await response.Content.ReadAsStringAsync();
     content.Should().StartWith($"# {rule.Id}: ");
 }
 ```
 
Things to consider:
1. The markdown is not longer available at `https://dotnet-project-file-analyzers.github.io`
2. The markdown files do not longer start with the tile (to facilitate the menu generation)
3. The HTML is open to change, and checking existence and title of the HTML is less trivial
